storehaus-algebra/src/test/scala/com/twitter/storehaus/algebra/ReadableStoreProperties.scala

```diff
@@ -16,17 +16,17 @@
 
 package com.twitter.storehaus.algebra
 
-import com.twitter.util.{ Await, Future }
+import com.twitter.util.Await
 import com.twitter.storehaus.ReadableStore
-import org.scalacheck.{ Arbitrary, Properties }
-import org.scalacheck.Gen.choose
+import org.scalacheck.{Prop, Arbitrary, Properties}
 import org.scalacheck.Prop._
 
-object ReadableStoreAlgebraProperties extends Properties("ReadableStoreAlgebra") {
+object ReadableStoreProperties extends Properties("ReadableStoreAlgebra") {
```

Comment:
The file and the class didnt have the same name so I picked one. I can definitely go the other way around though.